docs/assistant/use-attack-discovery-ai-assistant-incident-reporting.asciidoc

```diff
@@ -0,0 +1,65 @@
+
+After Attack discovery outlines your threat landscape, Elastic AI Assistant can help you quickly analyze a threat in detail. 
+
+[discrete]
+[[use-case-incident-reporting-use-ai-assistant-to-analyze-a-threat]]
+== Use AI Assistant to analyze a threat
+
+From a discovery on the Attack discovery page, you can click **View in AI Assistant** to start a chat that includes the discovery as context. 
+
+AI Assistant can quickly compile and present key data and provide suggestions to help you generate an incident report and plan an effective response. You can prompt it to supply relevant data or suggestions with questions like “How can I remediate this threat?” or “What ES|QL query would isolate actions taken by this user?” 
+
+image::images/attck-disc-esql-query-gen-example.png[An AI Assistant dialogue in which the user asks for a purpose-built {esql} query,90%]
+
+The screenshot above shows an {esql} query generated by AI Assistant in response to a user prompt. Learn more about <<esql-queries-assistant,using AI Assistant for {esql}>>.
+
+At any point in a conversation with AI Assistant, you can add data, narrative summaries, and other information from its responses to {elastic-sec}'s case management system to help quickly generate incident reports. 
+
+[discrete]
+[[use-case-incident-reporting-create-a-case-using-ai-assistant]]
+== Create a case using AI Assistant
+
+From the AI Assistant dialogue, click **Add to case** (image:images/icon-add-to-case.png[Add to case icon,19,16]) next to a message to add the information in that message to a case. If you add a message with an Attack discovery to a case, AI Assistant automatically adds the attack summary and all associated alerts to the case. You can also add AI Assistant messages containing remediation steps and relevant data to the case. 
+
+Cases help centralize relevant details in one place for easy sharing with stakeholders. Learn more about <<cases-overview,Cases>>. 
+
+[discrete]
+[[use-case-incident-reporting-translate]]
+== Translate incident information to a different human language using AI Assistant
+AI Assistant can translate its findings into other human languages, helping to enable collaboration within global security teams and distributed Security Operations Centers (SOCs), and making it easier to operate in multilingual organizations. 
+
+After AI Assistant provides information in one language, you can ask it to translate its responses. For example, if it provides remediation steps for an incident, you can instruct it to “Translate these remediation steps into Japanese.” You can then add the translated output to a case. This can help team members receive the same information and insights regardless of their primary language.
+
+In our internal testing, AI Assistant translations preserved the accuracy and utility of the original content. 
+
+
+[discrete]
+== Related documentation
```

Comment:
I'll let others weigh in on if they think this is necessary. Would be great as a sidebar component in our CSS styling, however :)

Reply:
I commented on the same type of section elsewhere (will link once I save this review).